src/reducers/transactionReducer.ts

```diff
@@ -58,6 +60,10 @@ export default function transactionReducer(state: TransactionState, action: Tran
       return { ...state, derivedReceiverAccount: action.payload.derivedReceiverAccount };
     case TransactionActionTypes.SET_GENERIC_RECEIVER_ACCOUNT:
       return { ...state, genericReceiverAccount: action.payload.genericReceiverAccount };
+    case TransactionActionTypes.SET_PAYLOAD:
+      return { ...state, payload: action.payload.payload };
+    case TransactionActionTypes.SET_ERROR:
+      return { ...state, error: action.payload.error };
```

Comment:
This sounds super generic - what kind of error is this? This reducer is doing a lot of things and it's really not clear which part this error is related to. If it's generic to *all* possible parameters then I'd argue it's not really a good choice, cause we most likely want a bit more fine-grained error handling.

Reply:
ok, i have followed your recommendation.

Then we need to work on a specific error component to display them all accordingly. 
The GH issue was already created: #176